host_usb_mixer_control/mixer_app.cpp

```diff
@@ -421,197 +438,224 @@ int main (int argc, char **argv) {
   unsigned int mixer_index = 0;
   unsigned int result = 0;
 
-
-
-  if (argc < 2) {
+  int min_argc;
```

Comment:
I guess you could accept the -g option on all platforms but only use it on Windows. This would remove some ifdef _WIN32 and make things a bit tidier maybe